commit a13366c632132bb9f8f2950a79773d8f68f4871e
Author: Adrian-Ken Rueegsegger <rueegsegger@swiss-it.ch>
Date:   Wed Jun 4 12:04:55 2008 -0700

    xfrm: xfrm_algo: correct usage of RIPEMD-160
    
    This patch fixes the usage of RIPEMD-160 in xfrm_algo which in turn
    allows hmac(rmd160) to be used as authentication mechanism in IPsec
    ESP and AH (see RFC 2857).
    
    Signed-off-by: Adrian-Ken Rueegsegger <rueegsegger@swiss-it.ch>
    Acked-by: Herbert Xu <herbert@gondor.apana.org.au>
    Signed-off-by: David S. Miller <davem@davemloft.net>

diff --git a/net/xfrm/xfrm_algo.c b/net/xfrm/xfrm_algo.c
index ac765dd9c7f5..23a2cc04b8cd 100644
--- a/net/xfrm/xfrm_algo.c
+++ b/net/xfrm/xfrm_algo.c
@@ -200,8 +200,8 @@ static struct xfrm_algo_desc aalg_list[] = {
 	}
 },
 {
-	.name = "hmac(ripemd160)",
-	.compat = "ripemd160",
+	.name = "hmac(rmd160)",
+	.compat = "rmd160",
 
 	.uinfo = {
 		.auth = {